src/pages/404/index.page.tsx

```diff
@@ -0,0 +1,37 @@
+import { Trans } from '@lingui/macro'
+import Link from 'antd/lib/typography/Link'
+import LanguageProvider from 'providers/LanguageProvider'
+import { useContext } from 'react'
+import { ThemeContext } from '../../contexts/themeContext'
+import error404 from '/public/assets/error404.png'
+
+const Custom404 = () => {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  return (
+    <LanguageProvider>
+      <div className="flex flex-col items-center justify-center p-6 md:p-40">
+        <img
+          src={error404.src}
+          alt="Image"
+          className="mb-6 h-60 w-60 object-center md:h-80 md:w-80"
+        />
+        <Trans>
+          <p className="mb-6 text-sm text-grey-600 md:text-base">
+            This page could not be found.
```

Comment:
**suggestion** perhaps we should `Trans` this line and the link text separately, wdyt?

Reply:
gotcha, changing! I gotta ask what is the difference? asking out of curiosity so I know for future implementations 🍻 

Is it so the translation pot breaks it into the next line for translation so it does not appear as one message which it isn't?

Reply:
@NakedFool yes exactly! Whether to seperate the strings is a case-by-case decision. In this case, I could imagine us using `Go to homepage` text in multiple spots, so we may as well only translate that _once_ and use the same translation everywhere. It's also totally valid to wrap the whole thing in a `Trans` too. Developer discretion!